pkg/networkmanager/firewall.jsx

```diff
@@ -760,7 +761,24 @@ class ActivateZoneModal extends React.Component {
                         <FormHelperText isHidden={false}>{_("The cockpit service is automatically included")}</FormHelperText>
                     </FormGroup>
 
-                    <FormGroup label={ _("Interfaces") } hasNoPaddingTop isInline>
+                    <FormGroup label={ _("Interfaces") }
+                               hasNoPaddingTop
+                               isInline
+                               labelIcon={
+                                   <Popover aria-label={_("Inerfaces help")}
+                                     enableFlip
+                                     bodyContent={
+                                         <Flex direction={{ default: 'column' }}>
+                                             <FlexItem>{_("Traffic passing through selected interfaces will respect this zone's settings.")}</FlexItem>
```

Comment:
Just making sure, you are explicitly ignoring the incoming vs. outgoing question here, right?

Honestly, I'm not sure what this sentence is trying to tell me, it feels rather tautological -- if I add an interface to a zone, then yes, I expect it to follow the rules of that zone.

It feels like it should explain what a "zone" is (essentially, a collection of interfaces and binding them to rules)? Perhaps

> This zone consists of the selected interfaces

But still, I find it rather hard to imagine why *this* part would be the confusion. If I look at that dialog, the big question mark that I have is what on earth a "Trust level" is. I.e. what kind of influence/behaviour change does it have when I change "public" to "external"? (It's an honest question -- I have no idea)